shard-operator-parent/shard-operator-core/src/main/resources/templates/observability-secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: smartevents-observability-configuration
+  namespace: smartevents-observability
+  labels:
+    configures: observability-operator
+    app.kubernetes.io/managed-by: bridge-fleet-shard-operator # used for filtering of Deployments created by the controller
```

Comment:
@manstis Could you align this with the `v2` operator name which is `smartevents-fleet-shard-operator`